blobfuse2-ci.yaml

```diff
@@ -51,6 +51,11 @@ jobs:
         sudo apt-get install $(fuselib) -y
       displayName: 'Install libfuse'
 
+    - script: |
```

Comment:
Do we need to run this here if we have it running on Workflows?

Reply:
workflows was not working earlier so thought of adding it here. Also the way that workflow gives output and this step gives is little different, so thought of adding it at both the places. But yes once workflow stablize we can just remove this part.